notebooks/community/model_garden/model_garden_tfvision_image_classification.ipynb

```diff
@@ -0,0 +1,771 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #23.    def upload_model(
</pre><p>This seems redundant, putting a wrapper around upload_model. Why not just initialize once and then just call method directly</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/vertex-ai-samples/pull/1625/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='10'/>

Reply:
<p>done</p>